README.md

```diff
@@ -77,24 +83,24 @@ Finally, scroll to the end of the page and hit **Save**.
-| <kbd>M-p p m</kbd> | `spotify-my-playlists`       | Show your playlists                             |
-| <kbd>M-p p f</kbd> | `spotify-featured-playlists` | Show the featured playlists                     |
-| <kbd>M-p p s</kbd> | `spotify-playlist-search`    | Search for playlists                            |
-| <kbd>M-p p u</kbd> | `spotify-user-playlists`     | Show playlists for the given user               |
-| <kbd>M-p p c</kbd> | `spotify-create-playlist`    | Create a new playlist                           |
-| <kbd>M-p t r</kbd> | `spotify-recently-played`    | List of recently played tracks                  |
-| <kbd>M-p t s</kbd> | `spotify-track-search`       | Search for tracks                               |
-| <kbd>M-p v u</kbd> | `spotify-volume-up`          | Increase the volume [2]                         |
-| <kbd>M-p v d</kbd> | `spotify-volume-down`        | Decrease the volume [2]                         |
-| <kbd>M-p v m</kbd> | `spotify-volume-mute-unmute` | Alternate volume between 0 and 100 [2]          |
-| <kbd>M-p M-d</kbd> | `spotify-select-device`      | Select a playback device [2]                    |
+| Key                  | Function                     | Description                                |
+|:---------------------|:-----------------------------|:-------------------------------------------|
+| <kbd>C-c . M-s</kbd> | `spotify-toggle-shuffle`     | Turn shuffle on/off [1]                    |
+| <kbd>C-c . M-r</kbd> | `spotify-toggle-repeat`      | Turn repeat on/off [1]                     |
+| <kbd>C-c . M-p</kbd> | `spotify-toggle-play`        | Play/pause                                 |
+| <kbd>C-c . M-f</kbd> | `spotify-next-track`         | Next track                                 |
+| <kbd>C-c . M-b</kbd> | `spotify-previous-track`     | Previous track                             |
+| <kbd>C-c . p m</kbd> | `spotify-my-playlists`       | Show your playlists                        |
+| <kbd>C-c . p f</kbd> | `spotify-featured-playlists` | Show the featured playlists                |
+| <kbd>C-c . p s</kbd> | `spotify-playlist-search`    | Search for playlists                       |
+| <kbd>C-c . p u</kbd> | `spotify-user-playlists`     | Show playlists for the given user          |
+| <kbd>C-c . p c</kbd> | `spotify-create-playlist`    | Create a new playlist                      |
+| <kbd>C-c . t r</kbd> | `spotify-recently-played`    | List of recently played tracks             |
+| <kbd>C-c . t s</kbd> | `spotify-track-search`       | Search for tracks                          |
+| <kbd>C-c . v u</kbd> | `spotify-volume-up`          | Increase the volume [2]                    |
+| <kbd>C-c . v d</kbd> | `spotify-volume-down`        | Decrease the volume [2]                    |
+| <kbd>C-c . v m</kbd> | `spotify-volume-mute-unmute` | Alternate the volume between 0 and 100 [2] |
+| <kbd>C-c . M-d</kbd> | `spotify-select-device`      | Select a playback device [2]               |
```

Comment:
According to the source file, the correct keybinding for this function would be `C-c . d`.

Reply:
Fixed!